utest/shared/src/main/scala/utest/runner/BaseRunner.scala

```diff
@@ -73,6 +74,8 @@ abstract class BaseRunner(val args: Array[String],
                eventHandler: EventHandler,
                taskDef: TaskDef) = {
 
+    startHeader foreach (h => println(h(path.fold("")(" " + _))))
```

Comment:
nitpick, `startHeader.foreach(...)`, infix should be limited to symbolic operators IMO

Reply:
Sure.